docs/spec-files/empty.md

```diff
@@ -4,13 +4,13 @@ Sometimes, only headers need to be communicated; there is no content for the bod
 
 The [PIURI](#protocol-identifier-uri) for this protocol is:
 
-    https://didcomm.org/empty/1.0
+    https://didcomm.org/empty/2.0
```

Comment:
empty is newly defined in this spec - so there's no 1.0 anywhere else.

Reply:
That makes sense.  When I saw the "1.0", I had associated it with the spec's version number instead of the protocol's version number.  I'm happy with whichever is most clear to everyone.